<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1301</width>
      <height>821</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Plot_Box"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>160</y>
      <width>222</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Button_Box">
     <item>
      <widget class="QPushButton" name="button_load">
       <property name="text">
        <string>Load Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_txt">
       <property name="text">
        <string>Export As txt(not implemented)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_threshold">
       <property name="text">
        <string>State Discrimination Threshold</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="text_threshold">
       <property name="text">
        <string>1.5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_apply">
       <property name="text">
        <string>Apply Threshold</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>30</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Radio_Box">
     <item>
      <widget class="QGroupBox" name="Radio_Group">
       <property name="title">
        <string>Data Represenation</string>
       </property>
       <widget class="QRadioButton" name="radio_counts">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Total Photon Counts</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio_qubitstate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>90</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Qubit State</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radio_avg_counts">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Average Photon Counts</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>no file loaded</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>340</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_sinefit">
       <property name="text">
        <string>Sinusoidal Fit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_fitampl">
         <property name="text">
          <string>Amplitude</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="text_fitampl"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_fitphase">
         <property name="text">
          <string>Phase</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="text_fitphase"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_fitfreq">
         <property name="text">
          <string>Freq(MHz)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="text_fitfreq"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_fitoffset">
         <property name="text">
          <string>Offset</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="text_fitoffset"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_sinefit">
       <property name="text">
        <string>FIT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_fitresults">
       <property name="text">
        <string>fit result will be displayed here</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>540</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_expfit">
       <property name="text">
        <string>Exponential Fit</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_fitampl2">
         <property name="text">
          <string>Amplitude</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="text_fitampl2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_fittimeconst">
         <property name="text">
          <string>Time Constant(us)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="text_fittimeconst"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_fitoffset2">
         <property name="text">
          <string>Offset</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="text_fitoffset2"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="button_expfit">
       <property name="text">
        <string>FIT(not implemented)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_fitresults_2">
       <property name="text">
        <string>fit result will be displayed here</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
